.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: required 
+dist: trusty 
```

Comment:
I'm following the example set in both `industrial_core` and [moveit](https://github.com/ros-planning/moveit_ci/blob/master/.travis.yml). I believe there is/was an issue with travis and 16.04. The build works with the current branch of our code.